inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -687,6 +718,7 @@ void MKLDNNGraph::Allocate() {
 void MKLDNNGraph::CreatePrimitives() {
     OV_ITT_SCOPED_TASK(itt::domains::MKLDNNPlugin, "MKLDNNGraph::CreatePrimitives");
```

Comment:
You inject scoped_task  in each loop iteration. No need it on function entry level.

Reply:
After discussion, we decided to keep this common counter. It is convenient to see a common group for multiple small counters for primitives creation.